src/pages/AddServers.js

```diff
@@ -52,9 +56,16 @@ class AddServers extends BaseUpdateWizardPage {
     //TODO implement
   }
 
+  //check if we can deploy the new servers
   isDeployable = () => {
-    //TODO implement
-    return false;
+    if(this.state.model && this.state.model.size > 0) {
```

Comment:
since `this.state.model` is set in the constructor, it seems like the check on the left of the && is unnecessary.

Reply:
That doesn't make any sense to me, but I'm not going to fret over a possibly extra check